src/content/docs/style-guide/formatting/code-examples.mdx

```diff
@@ -63,154 +65,173 @@ The following table provides some examples of our use of code format.
       <td>
         Method names
       </td>
-
+      <td>
+        Using code formatting for method names is standard practice.
+      </td>
       <td>
         To initialize the APM agent, use `startAgent()`.
       </td>
     </tr>
 
     <tr>
       <td>
-        Non-clickable URLs
+        URLs you don't intend to be clicked
+      </td>
+      <td>
+        Using code formatting hints to the user that they may need to customize or tweak the URL to ensure it works for them.
       </td>
-
       <td>
         In your web browser, navigate to the minion **Overview** page at `http://MINION_IP_ADDRESS`.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Command line utility names
+      </td>
+      <td>
+        If you think a reader or translator might confuse a command with a general English word, use code. 
 
-        The `<var>` formatting automatically applies color coding and italics in this situation.
+          * `curl`: Sends http requests via a terminal session–not to be confused with the curls you do with weights at the gym.
+          * `cat`: Lists the first ten lines of a file–not a feline pet who ignores you.
+          * `date`: Displays the day, year, and time–not an outing that couples take.
+          * `tail`: Displays the last ten lines of a file–not the appendage on various mammals.
+          * `which`: Show the location of a program executable–not the pronoun.
+
+      </td>
+      <td>
+        To install the utility, use `apt`.
       </td>
     </tr>
+
   </tbody>
 </table>
 
+## Inline code formatting [#inline-code]
+
+To surround small blocks of code or data (single words or lines), surround the text with backtick `` ` `` characters. 
```

Comment:
@austin-schaefer, I think "small blocks of code" is ambiguous. How about limiting inline usage to any code within a sentence. I think it's better to use the three backticks for a single line of code, which would be a code block (in the next section).

Reply:
Good call, and agree that using this for a line of code is undesirable. Rephrased and added a couple exmaples!